content/use-cases/use-an-escrow-as-a-smart-contract.md

```diff
@@ -0,0 +1,149 @@
+
+Because participants want to create a conditionally-held escrow to provide the smart contract, they need a condition value and a fulfillment value. In this scenario, the participant that creates these values is the neutral party planner (oracle).
+
+The party planner generates the condition and fulfillment values and provides the condition value to the party host, who creates the escrow.
+
+The party planner must keep the fulfillment value a secret. They will use the condition and fulfillment values to finish the escrow.
+
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+## [Calculate time values needed for the escrow](send-a-conditionally-held-escrow.html#2-calculate-release-or-cancel-time) (sender)
+
+Because the participants want the escrow to be eligible for cancellation after 12 noon the day after the party, the party host (sender) must calculate a `CancelAfter` value to include in the escrow definition.
+
+Because the participants want to disallow the escrow from being finished until after 11:00 pm the night of the party (the time when the party band is scheduled to finish its set), the party host must calculate a `FinishAfter` value to include in the escrow definition.
+
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+## [Create the escrow](send-a-conditionally-held-escrow.html#3-submit-escrowcreate-transaction) (sender)
+
+The party host (sender) creates the escrow that provides the smart contract. The party host must create the escrow because they are the only participant that can authorize the lock up and potential payout of XRP from their XRP Ledger account.
+
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+## [Wait for validation](send-a-conditionally-held-escrow.html#4-wait-for-validation) and [confirm escrow creation](send-a-conditionally-held-escrow.html#5-confirm-that-the-escrow-was-created) (sender and receiver)
+
+The party host (sender) waits for validation of the ledger that contains the escrow creation transaction and then confirms that the escrow was created.
+
+The party host then provides the escrow transaction's `hash` value to the party band (receiver). The party band can use the `hash` value to look up the escrow transaction on the XRP Ledger to ensure that it was created according to the smart contract terms they agreed to.
+
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+## [Finish the escrow](send-a-conditionally-held-escrow.html#6-submit-escrowfinish-transaction) (oracle)
+
+The party band (receiver) shows up and plays their set.
+
+The party planner (oracle) finishes the escrow using the fulfillment value they generated and kept secret.
+
+Alternatively, if the party band is a no-show, the party planner does not finish the escrow. In this case, after 12 noon the next day, any participant can [cancel the escrow](cancel-an-expired-escrow.html) to return the held XRP to the party host's (sender's) account.
```

Comment:
> the party planner does not finish the escrow

Change to:

> the party planner does not publish the fulfillment.

Also change "any participant can cancel..." to "anyone can cancel" because literally anyone can cancel an expired escrow.